test/e2e/utils/kubernetes.go

```diff
@@ -1040,3 +1040,23 @@ func (k TestKubernetes) WaitForValidRestorePhase(name, namespace string, phase i
 	}
 	ExpectNoError(err)
 }
+
+func (k TestKubernetes) GetDeployment(name, namespace string) *appsv1.Deployment {
+	deployment := &appsv1.Deployment{}
+	key := types.NamespacedName{
+		Namespace: namespace,
+		Name:      name,
+	}
+	ExpectMaybeNotFound(k.Kubernetes.Client.Get(context.TODO(), key, deployment))
+	return deployment
+}
+
+func (k TestKubernetes) GetConfigMap(name, namespace string) *corev1.ConfigMap {
```

Comment:
Uber nitpick, can you move this to reside next to the other `*ConfigMap` functions in the file.